config/v1/types_node.go

```diff
@@ -77,6 +77,7 @@ type NodeStatus struct {
 }
 
 // +kubebuilder:validation:Enum=v1;v2;""
+// +kubebuilder:validation:XValidation:rule="self != \"v1\"",message="cgroups v1 is not supported on openshift anymore"
```

Comment:
Why say openshift here? Is it supported by others? Why are we making a distinction, and why is "anymore" important?

Reply:
Updated the error message. Does it make sense now?